Content.Shared/ADT/Combat/Components/CombatModePickupWhitelistComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared.Whitelist;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.ADT.Combat;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class CombatModePickupWhitelistComponent : Component
+{
```

Comment:
```suggestion

/// <summary>
/// Компонент, ограничивающий возможность поднимать предметы, 
/// когда существо находится в боевом режиме.
/// 
/// Работает по системе whitelist/blacklist:
/// - Если задан <see cref="Whitelist"/>, поднимать можно только предметы из этого списка.
/// - Если задан <see cref="Blacklist"/>, поднимать нельзя предметы из этого списка.
/// </summary>
[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
public sealed partial class CombatModePickupWhitelistComponent : Component
{
```


Можно добавить документацию к компоненту